torch/_torch_docs.py

```diff
@@ -7385,6 +7385,45 @@ def merge_dicts(*dicts):
             [3, 2, 1, 0]])
 """.format(**common_args))
 
+add_docstr(torch.msort,
+           r"""
+msort(input, descending=False, *, out=None) -> (Tensor, LongTensor)
```

Comment:
This will need to be updated per the above comments

Reply:
Addressed!